tubemq-client-twins/tubemq-client-go/codec/codec.go

```diff
@@ -0,0 +1,107 @@
+package codec
```

Comment:
Need to add the LICENSE description of apache v2 to the header part of all files

Reply:
sorry, I forgot this. I will added the license.